frontend/awx/resources/templates/WorkflowVisualizer/wizard/NodeTypeStep.tsx

```diff
@@ -1,644 +0,0 @@
-          awxAPI`/job_templates/${template.id.toString()}/launch/`
-        );
-      } else if (
-        templateType === RESOURCE_TYPE.workflow_job ||
-        templateType === 'workflow_job_template'
-      ) {
-        launchConfigResults = await requestGet<LaunchConfiguration>(
-          awxAPI`/workflow_job_templates/${template.id.toString()}/launch/`
-        );
-      }
-      const { job_tags, skip_tags, inventory, ...defaults } = launchConfigResults.defaults;
-
-      launchConfigValue = {
-        ...defaults,
-        inventory: inventory?.id ? inventory : null,
-        job_tags: parseStringToTagArray(job_tags || ''),
-        skip_tags: parseStringToTagArray(skip_tags || ''),
-      };
-
-      const shouldShowPromptStep = !shouldHideOtherStep(launchConfigResults);
-      const shouldShowSurveyStep = launchConfigResults.survey_enabled;
-      if (shouldShowPromptStep || shouldShowSurveyStep) {
-        setWizardData((prev) => ({
-          ...prev,
-          launch_config: launchConfigResults,
-        }));
-        if (shouldShowPromptStep && shouldShowSurveyStep) {
-          setVisibleSteps(allSteps);
-        } else if (shouldShowPromptStep) {
-          const filteredSteps = allSteps.filter((step) => step.id !== 'survey');
-          setVisibleSteps(filteredSteps);
-        } else {
-          const filteredSteps = allSteps.filter((step) => step.id !== 'nodePromptsStep');
-          setVisibleSteps(filteredSteps);
-        }
-
-        if (stepData.nodePromptsStep && nodeResource) {
-          const { isDirty: isNodeTypeDirty } = getFieldState('node_type');
-          if (!isNodeTypeDirty && nodeResource.id === defaultValues?.node_resource?.id) {
-            setValue('prompt', { ...stepData.nodePromptsStep?.prompt });
-          } else {
-            // If the node type is not dirty and the node resource is not the same as the default value,
-            // and the wizard data is not the same as the default value, then reset the prompt to the default value
-            // else, set the prompt data to the current data.
-            setValue('prompt', launchConfigValue);
-            if ('type' in nodeResource && nodeResource.type !== templateType) {
-              setValue('node_resource', template);
-            }
-          }
-        }
-      } else {
-        const filteredSteps = allSteps.filter(
-          (step) => step.id !== 'nodePromptsStep' && step.id !== 'survey'
-        );
-        setVisibleSteps(filteredSteps);
-        setWizardData((prev) => ({ ...prev, launch_config: null }));
-      }
-    };
-
-    if (nodeType === RESOURCE_TYPE.job || nodeType === RESOURCE_TYPE.workflow_job) {
-      void setLaunchToWizardData();
-    }
-  }, [
-    allSteps,
-    defaultValues,
-    getFieldState,
-    getValues,
-    nodeResource,
-    nodeType,
-    setValue,
-    setVisibleSteps,
-    setWizardData,
-    stepData,
-  ]);
-
-  return (
-    <>
-      {pathname.split('/')[1] === 'schedules' ? (
```

Comment:
These checks are missing from this refactor and has removed conditional code that renders schedule specific inputs for the first step of the schedule wizard.

Reply:
This file has been moved, and broken down into smaller files.  It, and its parts now live in /common/SharedWizard 
Here is what that same view looks like on my branch for me.  
![Screenshot 2024-04-04 at 7 03 01 PM](https://github.com/ansible/ansible-ui/assets/39280967/b4e658b2-789e-4506-a8dc-b0c6fb9a97f7)

Reply:
When I open the schedule wizard from the template schedules page not rendering correctly:
![image](https://github.com/ansible/ansible-ui/assets/53712446/c9506b6b-f9cc-4742-a021-ad717b836eb0)